osgi.specs/docbook/159/service.feature.xml

```diff
@@ -729,6 +729,29 @@ Feature f = builder.build();</programlisting>
         
     </section>
   </section>
+
+  <section>
+    <title>Resource Versioning</title>
+
+    <para>
+    Feature JSON resources are versioned to support the updates the JSON structure in the future.
+    To declare the document version of the Feature use the <code>feature-resource-version</code>
+    key in the JSON document.
+    </para>
+      <programlisting>{
+  "feature-resource-version": "1.0",
+  "id": "org.acme:acmeapp:1.0.0"
+
+  /*
+    Additional Feature entities here
+    ...
+  */
+}</programlisting>
+    <para>
+    The currently supported version of the Feature JSON documents is 1.0. If no
+    Feature Resource Version is specified <code>1.0</code> is used as the default.
```

Comment:
For the example JSON documents in the spec, we should probably make sure they all include `"feature-resource-version": "1.0"` to encourage others to do so.

Reply:
I agree. However I would like to do this in a separate PR. There are a number of PRs open right now already and making that change would probably cause merge conflicts...

Reply:
As I had to rebase anyway I have now added the `"feature-resource-version"` to all the JSON snippets.